src/Compilers/Core/Portable/Operations/ControlFlowGraph.cs

```diff
@@ -95,6 +95,12 @@ public sealed partial class ControlFlowGraph
                             graph = ControlFlowGraphBuilder.Create(operation, enclosing);
                             break;
 
+                        case OperationKind.AnonymousFunction:
+                            var anonymousFunction = (IAnonymousFunctionOperation)operation;
```

Comment:
`anonymousFunction` is only used in `Debug.Assert`. Consider inlining in the `Assert` call. Same comment for `localFunction` above.